notebooks/00_quick_start/als_movielens.ipynb

```diff
@@ -13,11 +13,18 @@
    "cell_type": "markdown",
```

Comment:
I would suggest to add the same comment to ALS deep dive notebook https://github.com/Microsoft/Recommenders/blob/staging/notebooks/02_model/als_deep_dive.ipynb

Reply:
Added